### DISCUSSION TITLE
Authelia fatal msg="The following providers had fatal failures during startup:

### DISCUSSION THREAD

**Discussion by CodisTech** [authelia/authelia#6137] [Q&A] [answered]
Hello all,  I have been trying for a while to get authelia working.  After many attempts and different tutorials, trying to utilize the authelia documentation, i cannot figure it out. 

I would appreciate a guide in the right direction with my setup.  Below are the logs.  Please let me know if you require anything else.

`time="2023-10-15T11:55:51-07:00" level=info msg="Authelia v4.37.5 is starting"
time="2023-10-15T11:55:51-07:00" level=info msg="Log severity set to info"
time="2023-10-15T11:55:51-07:00" level=info msg="Storage schema is being checked for updates"
time="2023-10-15T11:55:51-07:00" level=info msg="Storage schema is already up to date"
time="2023-10-15T11:55:51-07:00" level=error msg="Failure running the user provider startup check: error reading the authentication database: could not parse the YAML database: yaml: unmarshal errors:\n  line 8: cannot unmarshal !!str `-admins...` into []string" stack="github.com/authelia/authelia/v4/internal/commands/root.go:287 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:87  cmdRootRun\ngithub.com/spf13/cobra@v1.6.1/command.go:920                  (*Command).execute\ngithub.com/spf13/cobra@v1.6.1/command.go:1044                 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.6.1/command.go:968                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1594                                      goexit"
time="2023-10-15T11:55:51-07:00" level=fatal msg="The following providers had fatal failures during startup: user" stack="github.com/authelia/authelia/v4/internal/commands/root.go:309 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:87  cmdRootRun\ngithub.com/spf13/cobra@v1.6.1/command.go:920                  (*Command).execute\ngithub.com/spf13/cobra@v1.6.1/command.go:1044                 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.6.1/command.go:968                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1594                                      goexit"`

**Comment by james-d-elliott** [comment, marked as answer]
You need to fix your user DB, here is an example: https://www.authelia.com/reference/guides/passwords/#yaml-format

Looks like you have `-admins` instead of `- admins`.

**Comment by CodisTech** [comment]
@james-d-elliott 

Thank you so much!  That was the issue! As im sure you know when you are looking at the same file for so long you miss the small things.  Much appreciated!

**Reply by james-d-elliott** [reply]
Enjoy!
